test/metabase/transforms/core_test.clj

```diff
@@ -0,0 +1,49 @@
+(ns metabase.transforms.core-test
+  (:require [expectations :refer [expect]]
+            [metabase.models
+             [card :refer [Card]]
+             [field :refer [Field]]
+             [table :as table :refer [Table]]]
+            [metabase.query-processor :as qp]
+            [metabase.test
+             [automagic-dashboards :refer [with-rasta]]
+             [data :as data]
+             [transforms :refer :all]
+             [util :as tu]]
+            [metabase.transforms.core :as t]))
+
+(expect
+  [:field-id (data/id :venues :price)]
+  (#'t/mbql-reference (Field (data/id :venues :price))))
+
+(expect
+  [:field-literal "PRICE" :type/Integer]
+  (#'t/mbql-reference (dissoc (Field (data/id :venues :price)) :id)))
+
+
+(expect
```

Comment:
The same comment as earlier applies here -- I don't think it's really a good enough test to just make sure the function returns something without throwing an Exception. Unless the function being tested is a predicate function, just checking for truthy values doesn't do a whole lot for us. Can we tweak this test to make sure the function returns the output we expect, given the input?

Reply:
I'll do this in 10437 which will also implement split loading from transforming (the relevant parts change so much it's pointless to do it here)